source/common/http/http1/codec_impl.h

```diff
@@ -201,6 +201,7 @@ class ConnectionImpl : public virtual Connection, protected Logger::Loggable<Log
   }
   uint32_t bufferLimit() { return connection_.bufferLimit(); }
   virtual bool supportsHttp10() { return false; }
+  virtual bool allowChunkedLength() { return false; }
```

Comment:
Any reason to allow this in one direction and not the other?

Reply:
+1 why not just always do this and/or pass the config option in? We have it available in both upstream and downstream?